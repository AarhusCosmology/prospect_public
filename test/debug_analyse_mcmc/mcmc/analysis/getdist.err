WARNING:root:outlier fraction 0.005706612957368244 
