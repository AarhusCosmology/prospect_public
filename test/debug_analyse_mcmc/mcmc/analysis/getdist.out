test/debug_analyse_mcmc/mcmc/test_2.txt
test/debug_analyse_mcmc/mcmc/test_3.txt
test/debug_analyse_mcmc/mcmc/test_1.txt
test/debug_analyse_mcmc/mcmc/test_0.txt
Removed no burn in
